src/caffeine/image_enumeration_m.f90

```diff
@@ -24,17 +24,17 @@ module function num_images_team_number(team_number) result(image_count)
 
   end interface
 
-  interface caf_this_image
+  interface prif_this_image
```

Comment:
`prif_this_image` in the draft spec has a very different generic subroutine interface (including different argument order). This is probably not the only conformance problem (even ignoring the unimplemented interfaces), just one I happened to notice.

I realize this PR is focused on renaming, do you want to do a separate pass for interface conformance?